processmanager/processinfo.go

```diff
@@ -548,9 +553,6 @@ func (pm *ProcessManager) ProcessPIDExit(pid libpf.PID) bool {
 				address, pid, err)
 		}
 	}
-	delete(pm.pidToProcessInfo, pid)
```

Comment:
This is now taking place in `ProcessedUntil`, delayed until `traceCaptureKTime >= pidExitKtime`.